source/frontend/src/features/mapSideBar/disposition/tabs/offersAndSale/OffersAndSaleContainerView.tsx

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import {
+  Api_DispositionFile,
+  Api_DispositionFileOffer,
+  Api_DispositionFileSale,
+} from '@/models/api/DispositionFile';
+import { prettyFormatDate } from '@/utils/dateUtils';
+import { formatMoney } from '@/utils/numberFormatUtils';
+
+import DispositionOfferDetails from './dispositionOffer/dispositionOfferDetails/DispositionOfferDetails';
+import DispositionSaleContactDetails from './dispositionOffer/dispositionSaleContactDetails/DispositionSaleContactDetails';
+
+export interface IOffersAndSaleContainerViewProps {
+  loading: boolean;
+  dispositionFile: Api_DispositionFile;
+  dispositionOffers: Api_DispositionFileOffer[];
+  dispositionSale: Api_DispositionFileSale | null;
+}
+
+const OffersAndSaleContainerView: React.FunctionComponent<IOffersAndSaleContainerViewProps> = ({
```

Comment:
nit containerview -> view
all containers have views, so no need to include container in the naming of a view.